readthedocs/search/documents.py

```diff
@@ -111,23 +111,40 @@ class Meta:
 
     def prepare_domains(self, html_file):
         """Prepares and returns the values for domains field."""
-        domains_qs = html_file.sphinx_domains.exclude(
-            domain='std',
-            type__in=['doc', 'label']
-        ).iterator()
-
-        all_domains = [
-            {
-                'role_name': domain.role_name,
-                'doc_name': domain.doc_name,
-                'anchor': domain.anchor,
-                'type_display': domain.type_display,
-                'doc_display': domain.doc_display,
-                'name': domain.name,
-                'display_name': domain.display_name if domain.display_name != '-' else '',
-            }
-            for domain in domains_qs
-        ]
+        all_domains = []
+
+        try:
+            domains_qs = html_file.sphinx_domains.exclude(
+                domain='std',
+                type__in=['doc', 'label']
+            ).iterator()
+
+            all_domains = [
+                {
+                    'role_name': domain.role_name,
+                    'doc_name': domain.doc_name,
+                    'anchor': domain.anchor,
+                    'type_display': domain.type_display,
+                    'doc_display': domain.doc_display,
+                    'name': domain.name,
+                    'display_name': domain.display_name if domain.display_name != '-' else '',
+                }
+                for domain in domains_qs
+            ]
+
+            log.debug("[%s] [%s] Domains for file %s are: %s" % (
+                html_file.project.slug,
+                html_file.version.slug,
+                html_file.path,
+                all_domains,
```

Comment:
I think we can probably just do a `len(all_domains)` to make the logs more readable.